app/ActivityListeners/PostLockHandler.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Created by IntelliJ IDEA.
+ * User: Duc Bob
+ * Date: 10/16/2017
+ * Time: 1:44 PM
+ */
+
+namespace App\ActivityListeners;
+
+
+use App\Models\Activity;
+use App\Models\ActivityMeta;
+use App\Models\Post;
+
+class PostLockHandler implements BaseHandler
+{
+    public static function event()
+    {
+        return [
+            'PostController@postLock',
+        ];
+    }
+    
+    public function handle(Activity $activity)
+    {
+        $mess = ' LOCKED!';
+        if(isset($activity->meta['LockError'][0])) {
+            $mess .= ' ERROR - ' . $activity->meta['LockError'][0];
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$mess". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS1488)